application/src/main/java/uk/nhs/hee/tis/revalidation/validator/TraineeRecommendationRecordDTOValidator.java

```diff
@@ -56,6 +57,12 @@ public void validate(Object target, Errors errors) {
           if (recordDTO.getDeferralDate() == null || recordDTO.getDeferralDate().isBefore(now())) {
             errors.reject("DeferralDate", "Deferral date can't be empty or in past");
           }
+          //Doctor X has a submission due date of 120 days from today (today <= 120 days)
+          if (!(recordDTO.getGmcSubmissionDate() != null
+              && (ChronoUnit.DAYS.between(now(), recordDTO.getGmcSubmissionDate())) <= 120)) {
```

Comment:
I would change this to remove the double negative.  There are some code smells related to the use of negation.

Reply:
`recordDTO.getGmcSubmissionDate() == null || (ChronoUnit.DAYS.between(now(), recordDTO.getGmcSubmissionDate())) <= 120)`